### DISCUSSION TITLE
Podman Instructions

### DISCUSSION THREAD

**Discussion by ctml91** [abraunegg/onedrive#1860] [General]
Podman is becoming more popular, especially in the Fedora space so it would be helpful to include instructions how to run this in podman because the startup seems to do a number of things it doesn't like with regards to permissions.

```
podman run -it --name onedrive -v onedrive-conf:/onedrive/conf -v "${ONEDRIVE_DATA_DIR}:/onedrive/data" -e "ONEDRIVE_UID=33" -e ONEDRIVE_GID=1000 --user 33:1000 driveone/onedrive:latest
+ : 33
+ : 1000
++ getent group 1000
+ odgroup=1000:x:1000:33
+ odgroup=1000
++ getent passwd 33
+ oduser='33:*:33:1000:container user:/usr/src/onedrive:/bin/sh'
+ oduser=33
+ usermod -g 1000 33
usermod: no changes
+ grep -qv root /dev/fd/63
++ groups 33
+ chown 33:1000 /onedrive/ /onedrive/conf
chown: changing ownership of '/onedrive/': Operation not permitted
chown: changing ownership of '/onedrive/conf': Permission denied
```

**Comment by abraunegg** [comment]
@ctml91 
> ```
> + chown 33:1000 /onedrive/ /onedrive/conf
> chown: changing ownership of '/onedrive/': Operation not permitted
> ....
>```

This is an interesting issue - which should be fixed by #1861

However, I do not have the time to test and deploy and write documentation as to how to get the client working on each an every platform everyone tries to use.

Please can you see if firstly #1861 resolves your issue - but then more importantly if this does, please document how you are running it in podman - create a document and submit that a PR for inclusion to the repository.

**Reply by ctml91** [reply]
Thank you @abraunegg. I have it running in podman with some minor changes to the entrypoint and podman run arguments used. I can submit a PR with the updated documentation, would you prefer podman instructions to be located separately in podman.md or included in the docker docs?

**Reply by abraunegg** [reply]
@ctml91 
Please if can you submit a PR with those changes and any specific podman instructions in a new document that would be greatly appreciated.

**Comment by abraunegg** [comment]
@ctml91 
Any update re a PR ?

It would be great to get this prior to starting to prep the v2.4.16 release

**Reply by ctml91** [reply]
Thanks for the reminder. Submitted

https://github.com/abraunegg/onedrive/pull/1871
